cg/apps/janus/api.py

```diff
@@ -0,0 +1,28 @@
+"""The JanusAPIClient."""
+
+from http import HTTPStatus
+
+import requests
+from requests import Response
+
+from cg.apps.janus.dto.create_collect_qc_request import CreateCollectQCRequest
+from cg.exc import JanusAPIError
+
+
+class JanusAPIClient:
+    def __init__(self, config: dict[str]):
+        self.host = config["janus"]["host"]
+
+    def get_qc_metrics(self, collect_qc_request: CreateCollectQCRequest):
+        endpoint: str = f"{self.host}/collect_qc_metrics"
+        response = requests.post(endpoint, data=collect_qc_request)
+        if response.status_code == HTTPStatus.OK:
+            return response.json()
```

Comment:
Parse this response json into a pydantic model and return that instead

Reply:
Yes, we do not want the clients to return a json string. It really needs to be parsed into a model

Reply:
So is the flow like 
- cg collects raw metrics from pipelines
- cg sends raw metrics to janus
- arnold retrieves metrics from janus
Or did I miss something?

Reply:
Arnold does not know when and what metrics to retrieve, that is handled by cg in conjunction with trailblazer and housekeeper. I thought about that as well, but I think we want to let cg handle communications between these things.